packages/components/src/navigation/group/index.js

```diff
@@ -6,9 +6,16 @@ import classnames from 'classnames';
 /**
  * Internal dependencies
  */
-import { GroupTitleUI } from './styles/navigation-styles';
+import { GroupTitleUI } from '../styles/navigation-styles';
+import { useNavigationMenuContext } from '../menu/context';
 
 export default function NavigationGroup( { children, className, title } ) {
+	const { isActive } = useNavigationMenuContext();
+
+	if ( ! isActive ) {
+		return children;
+	}
```

Comment:
I think we should leave a comment to explain why we are rendering `children` instead of `null`.
Here, in `Menu` and `Item` as well.

Something like:
`// Keep the children rendered to make sure inactive items are included in the navigation tree.`

Reply:
Done